### PR TITLE
Fixes Hypospray Theft Objective

### LINKED ISSUE #5232: Advanced Pinpointer points you to the nearest emergency autoinjector when set to hypospray
More futzing about with traitor items on my private server.

This one takes the cake, I think emergency autoinjector must be a child of the hypospray, because when you set the advanced pinpointer up to locate it, I was directed to a medkit box that had an emergency autoinjector in it. Sure enough, taking it out and taking it with me, the advanced pinpointer showed that I had the hypospray.

### PR DESCRIPTION
**Pull request by Kyep**
Bug:
- Currently, advanced pinpointers inherit steal objectives as potential targets. This means they can start tracking other subtypes of /obj/item/weapon/reagent_containers/hypospray - such as autoinjectors.

This PR:
- This PR modifies the theft objective for the hypospray, such that it always targets the unique CMO hypospray.

:cl: Kyep
fix: Fixed bug with adv pinpointers and CMO hypospray theft objectives. Fixes #5232 
/:cl:

### CODE REVIEW

**Comment by Kyep** [comment]
https://github.com/ParadiseSS13/Paradise/issues/5232

**Comment by mrturkeytoe** [comment]
Unrelated, but it would be neat if the adv pinpointer could track the nukes.

**Comment by monster860** [comment]
Don't forget to remove the snowflaked code here:

https://github.com/Kyep/Paradise/blob/9b317be92e069131549e18a287fbe09c47d2a9bb/code/game/gamemodes/steal_items.dm#L22-L24

**Comment by Kyep** [comment]
@monster860 deleted
@mrturkeytoe I'm not sure I want to do that, could have balance implications. Syndie nuke, regular nuke during DS, etc.

File: code/game/gamemodes/steal_items.dm [hunk 1]
```diff
@@ -111,7 +108,7 @@ datum/theft_objective/ai/check_special_completion(var/obj/item/device/aicard/C)
 
 /datum/theft_objective/hypospray
 	name = "a hypospray"
```
Comment (monster860) [inline code/game/gamemodes/steal_items.dm]: Might as well make this say "the CMO's hypospray"

**Comment by TheDZD** [comment]
You need to add one of the reserved words [here](https://help.github.com/articles/closing-issues-via-commit-messages/) followed by the issue number to the PR description for it to automatically close. Most often-used one is just "Fixes #ABCD"

**Comment by mrturkeytoe** [comment]
@Kyep But if a min runs a capture the nuke event and someone hides it in some obscure place/some jackass cheeses the nukie shuttle turrets and spaces their nuke they can still find it.

**Pull request merged**